dist/backdraft.js

```diff
@@ -1252,7 +1257,10 @@ _.extend(Plugin.factory, {
           self._columnManager.columnsSwapped(fromIndex, toIndex);
           // pass event up
           self._onColumnReorder();
-        }
+        },
+        // iFixedColumns configures how many columns should be unmovable starting from left
+        // if the first column is the bulk column we make it unmovable
+        iFixedColumns: this.$el.find("th:first.bulk :checkbox").length
```

Comment:
fair enough.. what about making the `th.first.bulk` a "constant", ie `this.BULK_COLUMN_SELECTOR`  and using throughout?